src/Foundatio.RabbitMQ/Messaging/RabbitMQMessageBus.cs

```diff
@@ -105,7 +119,7 @@ public class RabbitMQMessageBus : MessageBusBase, IMessageBus {
         /// </summary>
         /// <param name="messageType"></param>
         /// <param name="message"></param>
-        /// <param name="delay"></param>
+        /// <param name="delay">Along with the delay value, _delayExchange should also be set to true</param>
```

Comment:
This shouldn't be required as we do have a base class to do delayed messages (in process) while not as good or reliable, this is an implementation detail :)